parsons/zoom/zoom.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, account_id=None, client_id=None, client_secret=None):
             authorization_kwargs={"account_id": self.account_id},
         )
 
-    def _get_request(self, endpoint, data_key, params=None, **kwargs):
+    def _get_request(self, endpoint, data_key=None, params=None, **kwargs):
```

Comment:
what's the rationale behind defaulting this to `None` type?

Reply:
This method that I have returns data that doesn't need to be parsed further using a key.

Reply:
But I guess it doesn't necessarily need to be a default value